servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpConnector.java

```diff
@@ -141,6 +141,7 @@ protected void initChannel(Channel channel) {
                     public void connect(ChannelHandlerContext ctx, SocketAddress remoteAddress,
                                         SocketAddress localAddress, ChannelPromise promise) throws Exception {
                         ctx.pipeline().remove(this);
+                        super.connect(ctx, remoteAddress, localAddress, promise);
                         promise.addListener(f -> {
```

Comment:
Consider adding a comment, clarifying why we need to call `super` before adding a listener

Reply:
It turns out it's no longer necessary so I reverted it.